store/index.js

```diff
@@ -1,114 +1,116 @@
 // Root state - Stores state data
 export const state = () => ({
 
-	pageNames: {
-		
-		home: {
-			fullName: "Home",
-			location: "/",
-			pageName: "index",
-		},
-
-		categorization: {
-			fullName: "Categorization",
-			location: "categorization",
-			pageName: "categorization"
-		},
-
-		annotation: {
-			fullName: "Annotation",
-			location: "annotation",
-			pageName: "annotation"
-		},
-
-		download: {
-			fullName: "Download",
-			location: "download",
-			pageName: "download"
-		}
-	},
+	// Page-related data
 
 	pageData: {
 
-		// index.vue
-		home: {
-
-			// Participants.tsv file data
-			tsvFile: null,
-
-			// Data dictionary file data
-			jsonFile: null
-		},
-
-		// categorization.vue
-		categorization: {
-
-			// Category paint brushes
-			current: {
-	
-				bColor: "white",
-				category: "",
-				fColor: "black"
-			},
-			default: {
-	
-				bColor: "white",
-				category: "",
-				fColor: "black"			
-			},
-
-			// Stores currently painted items
-			paintingData: {},
-
-			// Stores currently loaded table data
-			tableData: {}
-		},
-
-		// annotation.vue
-		annotation: {
-
-		},
-
-		// download.vue
-		download: {
-
-		}
-	},
-
-	painting: {
-		
-		classes: {
-
-			paint0: "category-style-0",
-			paint1: "category-style-1",
-			paint2: "category-style-2",
-			paint3: "category-style-3",
-			paint4: "category-style-4"
-		},
-
-		palette: {
-			backgroundColors: [],
-			foregroundColors: []
-		}
-	}
+        home: {
+            
+            accessibility: true,
+            fullName: "Home",
+            location: "/",
+            pageName: "index",
+        },
+
+        categorization: {
+            
+            accessibility: false,
+            fullName: "Categorization",
+            location: "categorization",
+            pageName: "categorization"
+        },
+
+        annotation: {
+            
+            accessibility: false,
+            fullName: "Annotation",
+            location: "annotation",
+            pageName: "annotation"
+        },
+
+        download: {
+            
+            accessibility: false,
+            fullName: "Download",
+            location: "download",
+            pageName: "download"
+        }
+    },
+
+    // TSV file
+
+    // Participants.tsv file data
+    // For format see 'convertTsvLinesToDict' in index.js
+    tsvDataOriginal: null,
```

Comment:
how about `rawDataTable`

Reply:
I was thinking that these `tsvData` fields in the store should be grouped under one object. How about something like:

```
tsvData: {
    raw: null,
    annotated: []
}
```

And a similar object for the `dataDictionary` fields. I'll just mention this here instead of replying to each of your comments on the store names.

Reply:
oh, yeah, I think that's a better way of handling this, agreed! In this case I would still drop the file extension from the variable name. Maybe `dataTable` and `dataDictionary`?

Reply:
Sounds good

Reply:
Just an addendum: How about `dataTable.original` instead of `dataTable.raw`? Raw data is not actually stored here as it's converted into the format (array of objects) asked for by `b-table`?